frontend/packages/client/src/components/Community/CommunityPropsAndVoting.js

```diff
@@ -0,0 +1,186 @@
+        style={{
+          display: "flex",
+          justifyContent: "flex-end",
+          alignContent: "center",
+          position: "absolute",
+          right: 15,
+          top: 9,
+        }}
+      >
+        {enableDelete && (
+          <div
+            className="cursor-pointer is-flex is-align-items-center"
+            onClick={() => onDeleteStrategy(index)}
+          >
+            <Bin />
+          </div>
+        )}
+      </div>
+    </div>
+  );
+};
+
+export default function CommunityProposalsAndVoting({
+  communityVotingStrategies = [],
+  communityId,
+  updateCommunity,
+  updatingCommunity,
+} = {}) {
+  const { data: allVotingStrategies, loading: loadingAllStrategies } =
+    useVotingStrategies();
+
+  const [currentStrategies, setCurrentStrategies] = useState(
+    communityVotingStrategies.map((st) => ({ strategy: st, fromServer: true }))
+  );
+
+  // holds array of objects with strategy information to be added
+  const [newStrategies, setNewStrategies] = useState([]);
+
+  const { openModal, closeModal } = useModalContext();
+
+  // filter strategis: remove existing on community
+  // and the ones to be added sent to add in the backend
+  const strategiesToAdd = (allVotingStrategies || []).filter(
+    (st) =>
+      ![...currentStrategies, ...newStrategies].find(
+        (currentSt) =>
+          currentSt.strategy === st.key && currentSt?.toBeremoved !== true
+      )
+  );
+
+  const addNewStrategy = (newStrategyInfo) => {
+    setNewStrategies((state) => [...state, newStrategyInfo]);
+    closeModal();
+  };
+
+  const onAddStrategy = () => {
+    openModal(
+      <StrategyEditorModal
+        onDismiss={() => closeModal()}
+        strategies={strategiesToAdd}
+        onDone={addNewStrategy}
+      />,
+      {
+        classNameModalContent: "rounded-sm",
+        showCloseButton: false,
+      }
+    );
+  };
+
+  // removes existing strategies on the community
+  const onDeleteStrategy = (index) => {
+    setCurrentStrategies((state) =>
+      state.map((datum, idx) => {
+        if (idx === index) {
+          return { ...datum, toBeRemoved: true };
+        }
+        return datum;
+      })
+    );
+  };
+
+  // removes new strategies not saved yet
+  const onDeleteNewStrategy = (index) => {
+    setNewStrategies((state) => state.filter((st, idx) => idx !== index));
+  };
+
+  // sends updates to backend
+  const saveData = () => {
+    // we need to remove strategies first:
+    // in case user removes and old one and
+    // creates the same strategy with different contract information
+    // use same signature for both requests
+    console.log(
+      "--- strategies to delete ---",
+      currentStrategies.filter((st) => st?.toBeremoved)
+    );
+    // we will add new strategies
+    console.log("--- strategies to add ---", newStrategies);
+
+    // await updateCommunity()
+  };
+
+  const savingData = false;
+
+  const enableDelete = currentStrategies.length + newStrategies.length > 1;
```

Comment:
we need at least one strategy, so we hide the delete icon when there's only one strategy

<img width="387" alt="Screen Shot 2022-06-17 at 21 32 18" src="https://user-images.githubusercontent.com/7526740/174415191-2a5d24e9-e862-4086-be05-a11d02f68982.png">